mods/gen4/formats-data.js

```diff
@@ -1588,7 +1588,7 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	glalie: {
-		randomBattleMoves: ["spikes", "icebeam", "iceshard", "earthquake", "explosion", "taunt"],
```

Comment:
Also why not keep Ice Shard? These are simply questions tbh I suppose, seeing as I feel they were viable